Permitted deprecation warning:

https://github.com/DataBiosphere/azul/blob/37ea947543ac8ca156b9dc3f53899ca8ed9e3b7c/test/azul_test_case.py#L150-L152

… cannot be removed yet due to use of `datetime.utcnow()` in Azul dependency `aws_requests_auth`:

```
======================================================================
ERROR: tearDownClass (integration_test.IndexingIntegrationTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/daniel/repo/azul3/test/azul_test_case.py", line 197, in tearDownClass
    assert False, list(map(str, caught_warnings))
           ^^^^^
AssertionError: ["{message : DeprecationWarning('datetime.datetime.utcnow() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.now(datetime.UTC).'), category : 'DeprecationWarning', filename : '/Users/daniel/repo/azul3/.venv/lib/python3.14/site-packages/aws_requests_auth/aws_auth.py', lineno : 107, line : None}"]
```

--- a/scripts/sell_unused_slots.py
+++ b/scripts/sell_unused_slots.py
@@ -6,6 +6,7 @@ import argparse
 from datetime import (
     datetime,
     timedelta,
+    timezone,
 )
 import logging
 import sys
@@ -79,9 +80,7 @@ class ReindexDetector:
         ]
 
     def _lambda_invocation_counts(self) -> dict[LambdaFunction, int]:
-        # FIXME: DeprecationWarning for datetime methods in Python 3.12
-        #        https://github.com/DataBiosphere/azul/issues/5953
-        end = datetime.utcnow()
+        end = datetime.now(timezone.utc)
         start = end - timedelta(minutes=self.interval)
         lambdas_by_name = {
             lambda_.name: lambda_ for lambda_ in self._list_contribution_lambda_functions()

--- a/src/azul/dss.py
+++ b/src/azul/dss.py
@@ -1,5 +1,6 @@
 from datetime import (
     datetime,
+    timezone,
 )
 import logging
 
@@ -9,9 +10,7 @@ version_format = '%Y-%m-%dT%H%M%S.%fZ'
 
 
 def new_version():
-    # FIXME: DeprecationWarning for datetime methods in Python 3.12
-    #        https://github.com/DataBiosphere/azul/issues/5953
-    return datetime.utcnow().strftime(version_format)
+    return datetime.now(timezone.utc).strftime(version_format)
 
 
 def validate_version(version: str):

